production/helm/loki/values.yaml

```diff
@@ -974,11 +974,11 @@ gateway:
     type: RollingUpdate
   image:
     # -- The Docker registry for the gateway image
-    registry: docker.io
+    registry: ghcr.io
```

Comment:
We use `docker.io` for everything else, as well as the other "grafana labs supported" helm charts (ie. `mimir-distributed`). This is also where we publish our images, and link to in documentation. I would like to maintain that consistency and don't think we should make this change.

Reply:
It's pretty unfortunate Grafana is so engrained to dockerhub given their 100/pulls per 6 hours rate limits and image retention nonsense. I'd love to see Grafana ditch dockerhub overall for quay, or GHCR or anything else that's more user friendly.

I know this is neither here nor there but if I worked for Grafana I'd make it my grand mission to migrate away from dockerhub for all projects, they are not friendly to the small guys anymore.

It's not great UX spinning up a fresh Kubernetes cluster and seeing logging or monitoring not working due to rate limits. Sure this can be solved with a pull thru cache, but Grafana could solve the problem for everyone by just migrating away from dockerhub. Sorry for my rant, it's just a passion of mine to see people migrate away from Dockerhub.